Content.Shared/GameObjects/Components/Inventory/EquipmentSlotDefinitions.cs

```diff
@@ -47,6 +47,8 @@ public enum Slots
             POCKET4,
             EXOSUITSLOT1,
             EXOSUITSLOT2,
+            LHAND,
+            RHAND,
```

Comment:
This is WAY too hardcoded... Currently you can add hands to any humans with a simple command, so it'd be best if you didn't hardcode slots like this, and instead handled multiple hands sanely.